prboom2/src/umapinfo.cpp

```diff
@@ -191,6 +191,114 @@ static const char * const ActorNames[] =
 	"PlasmaBall2",
 	"EvilSceptre",
 	"UnholyBible",
+	// Crispy/Retro
+	"MusicChanger",
```

Comment:
There shouldn't be anything port-specific in this file I think. Incidentally this entity is in pr+ too, so I don't think the crispy / retro comment makes sense.

MT_GIBDTH, which is 145, is used in pr+ as well.

The other comments about blood and rocket trail I would remove as well...these have entries in pr+'s info.h related to stealth creatures for instance, they have no defined cross-port meaning, or meaning in the standard then as far as I understand it.

Reply:
Comments have been removed, save the one noting where the 100 DEHEXTRA slots begin.